### PR TITLE
Review one of your past projects

### PR DESCRIPTION
**Pull request by sergio-usma**
I have reviewed all the semantic HTML and completed all the requirements of first project activity form week 2.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,151 +1,210 @@
...
-  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
-  <script src="https://cdn.tailwindcss.com"></script>
-</head>
-
-<body class="bg-gray-200">
-  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
-  <header id="header" class="flex justify-between px-24 py-6 fixed top-0 left-0 right-0">
-    <div class="w-64">
-      <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png" id="header-img"
-        width="100%" height="auto" />
```
Comment (sergio-usma) [inline index.html:21]: I widely recommend to don't using styles in the HTML code! It's a good practice to have the styles in different files, in order to keep the code organized and get a better SEO.

File: index.html [hunk 2]
```diff
@@ -1,151 +1,210 @@
...
-    </div>
+  <body class="bg-gray-200">
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    
+    <header
+      id="header"
+      class="flex justify-between px-24 py-6 fixed top-0 left-0 right-0"
```
Comment (sergio-usma) [inline index.html:27]: Please see comment No. 1 about the use of styles in the HTML file!

File: index.html [hunk 3]
```diff
@@ -1,151 +1,210 @@
...
+    >
+      <div class="w-64">
+        <img
+          src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png"
+          id="header-img"
+          width="100%"
+          height="auto"
```
Comment (sergio-usma) [inline index.html:34]: Please see comment No. 1 about the use of styles in the HTML file!

File: index.html [hunk 4]
```diff
@@ -1,151 +1,210 @@
...
+        />
+      </div>
 
-    <nav id="nav-bar" class="flex">
-      <ul class="flex gap-x-12">
```
Comment (sergio-usma) [inline index.html:25]: Please see comment No. 1 about the use of styles in the HTML file!

File: index.html [hunk 5]
```diff
@@ -1,151 +1,210 @@
...
-        <li><a class="nav-link" href="#features">Features</a></li>
-        <li><a class="nav-link" href="#hiw">How It Works</a></li>
-        <li><a class="nav-link" href="#pricing">Pricing</a></li>
-      </ul>
-    </nav>
-  </header>
+      <nav id="nav-bar" class="flex">
+        <ul class="flex gap-x-12">
```
Comment (sergio-usma) [inline index.html:39]: Please see comment No. 1 about the use of styles in the HTML file!

File: index.html [hunk 6]
```diff
@@ -1,151 +1,210 @@
...
+          <li><a class="nav-link" href="#features">Features</a></li>
+          <li><a class="nav-link" href="#hiw">How It Works</a></li>
+          <li><a class="nav-link" href="#pricing">Pricing</a></li>
+        </ul>
+      </nav>
+    </header>
 
-  <section id="hero" class="flex flex-col items-center mt-12 mb-24">
-    <h2 class="text-2xl font-bold">Handcrafted, home-made masterpieces</h2>
```
Comment (sergio-usma) [inline index.html:34]: Please see comment No. 1 about the use of styles in the HTML file!

File: index.html [hunk 7]
```diff
@@ -1,151 +1,210 @@
...
+      href="https://use.fontawesome.com/releases/v5.8.1/css/all.css"
+      integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf"
+      crossorigin="anonymous"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
```
Comment (sergio-usma) [inline index.html:18]: I kindly recommend adding the tag <meta name="viewport" content="width=device-width, initial-scale=1.0">, so it helps you to adjust a webpage's content to fit different devices and screen sizes. The "width=device-width" property matches the page width to the device screen width for an optimal user experience. The "initial-scale=1.0" property sets the initial zoom level to 100% for readability and navigation on mobile devices.

File: index.html [hunk 8]
```diff
@@ -1,151 +1,210 @@
...
-</body>
+        <div class="copyright text-right">
+          <span class="text-sm text-gray-600"
+            >Copyright 2016, Original Trombones</span
+          >
+        </div>
+      </footer>
+    </div>
 
-</html>
```
Comment (sergio-usma) [inline index.html:151]: I strongly recommend separating the styles into a separate CSS files. Besides, it's a good practice, helps your code to be more organized, and for example, implementing methodologies or techniques such as BEM https://getbem.com/introduction/ to look your code more stylish!

**Review by sergio-usma — COMMENTED** [review]
### CHANGES REQUIRED! ♻️ ### 
tenor

The code was functional and worked for the first trial project! Check the comments and recommend separating the styles into a CSS file for a more organized and stylish code. So, implementing good practices in HTML & CSS is important so other partners can understand and collaborate in your code!!!

**Review by braincee — APPROVED** [review]
Hi @sergio-usma 👋 , I am @braincee, your Code Reviewer.

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

youre-approved-shirley-auto-group

HIGHLIGHTS
------

- [x] Great implementation of inline comments 🤝 
- [ ] Great use of effective communication with politeness 🤝 

OPTIONAL CHANGES
---

- [ ] [OPTIONAL] I highly recommend adding emojis to your inline comments to look more human-friendly 😃.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @braincee._

---

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**